plugins/check_service.cpp

```diff
@@ -126,9 +126,7 @@ INT parseArguments(INT ac, WCHAR **av, po::variables_map& vm, printInfoStruct& p
 
 	printInfo.service = vm["service"].as<std::wstring>();
 
-	if (vm.count("warn")) {
-		printInfo.warn = true;
-	}
+	vm.count("warn") ? printInfo.warn = true : printInfo.warn = false;
```

Comment:
printInfo = vm.count("warn")?

Reply:
I'll guess you mean `printInfo.warn = vm.count("warn");`? ;)

I also tested this variant and it worked as expected. I would stick to this variant, as it "feels" somehow cleaner.